src/fragments/lib-v1/graphqlapi/flutter/mutate-data.mdx

```diff
@@ -1,11 +1,23 @@
-## Run a mutation
+In this guide, you will learn how to create, update, and delete your data using Amplify Libraries' GraphQL client.
```

Comment:
I think we can revert most of the changes in this file just to stay aligned with iOS/Android in vPrevious. If you want to leave the print->safePrint changes that is fine, but I would revert the rest.